services/src/main/java/org/keycloak/services/resources/admin/UsersResource.java

```diff
@@ -175,7 +175,7 @@ public Response createUser(final UserRepresentation rep) {
             if (session.getTransactionManager().isActive()) {
                 session.getTransactionManager().setRollbackOnly();
             }
-            throw ErrorResponse.error("Password policy not met", Response.Status.BAD_REQUEST);
+            throw ErrorResponse.error("Password policy not met", new Object[] { e.getMessage() }, Response.Status.BAD_REQUEST);
```

Comment:
Same comment like above, it would be good to just log this instead of adding into ErrorResponse